lock-api-objects/src/main/java/com/palantir/lock/watch/NoOpLockWatchEventCache.java

```diff
@@ -55,10 +52,19 @@ public CommitUpdate getCommitUpdate(long startTs) {
     @Override
     public TransactionsLockWatchEvents getEventsForTransactions(Set<Long> startTimestamps,
             Optional<IdentifiedVersion> version) {
-        return NONE;
+        IdentifiedVersion fakeVersion = generateFakeVersion();
```

Comment:
This now needs to generate a new random UUID on each call to `getEventsForTransactions` - is that intentional? I thought `-1` as a sequence number was enough to indicate that the Events value was meaningless.

Reply:
This was a suggestion from the other PR by Jakub (ensuring that it 100% is treated as a snapshot each time) but in honesty this is unnecessary - things go off the snapshot flag anyway.